todo_app/tests_e2e/test_client_e2e.py

```diff
@@ -0,0 +1,163 @@
+import os
+import pytest
+import requests
+import json
+from threading import Thread
+from time import sleep
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Load our real environment variables
+    #file_path = find_dotenv('.env')
+    file_path = find_dotenv('.env')
+    #load_dotenv(file_path,override=True)
+    load_dotenv(override=True)
```

Comment:
Currently this has some confusing different combos of code/comment, and fundamentally all it's doing is one command which defaults to finding `.env` anyway, so we could just have:
```suggestion
    load_dotenv(override=True)
```